for_review/must_gather.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import logging
+import os
+import shlex
+
+from ocp_utilities.infra import run_command
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def create_must_gather_command(
+    dest_dir,
+    image_url,
+    script_name=None,
+    kubeconfig=None,
+):
+    base_command = (
+        f"oc adm must-gather {f'--kubeconfig {kubeconfig}' if kubeconfig else ''} --image={image_url}"
```

Comment:
Should the kubeconfig be read from environment variable, if not passed?

Reply:
No, we should not "hide" it (mainly when working in multi-cluster env)